gitbooks/developing/building-rust-core.zh-CN.md

```
@@ -0,0 +1,190 @@
+这些包的重要性：
+
+- `build-essential` / `base-devel`、`cmake`、`pkg-config` / `pkgconf`：传递性 Rust 依赖使用的原生构建。
+- `clang`、`libclang-dev`：bindgen / C 和 C++ 编译路径，被原生 crate 使用。
+- `libssl-dev` / `openssl`：某些网络依赖需要的 OpenSSL 头文件。
+- `libasound2-dev` / `alsa-lib`、`libxi-dev` / `libxi`、`libxtst-dev` / `libxtst`、`libxdo-dev` / `xdotool`、`libudev-dev`（Arch 中已包含在 `systemd-libs` 内）、`libevdev`：被核心构建引入的音频/输入/设备 crate 所需。
+
+### `whisper-rs` + `clang` 注意事项
+
+`whisper-rs-sys` 在 `clang` 下可能会失败并提示：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)